test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1228,6 +1228,43 @@ UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_unified, rc, "rc")
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_unified, ud, "ud")
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_unified, rc_dc, "rc,dc")
 
+class select_ep_transport : public test_ucp_wireup {
```

Comment:
I think we could use already existing `test_ucp_wireup` test suite and add the new test there